TensorFlowV2_CNN_MNIST/README.md

```diff
@@ -0,0 +1,62 @@
+# TensorFlow MNIST
+
+This example shows distributed training of a simple TensorFlow Keras model using the MNIST dataset.
+Two approaches to distributed trainig are presented:
+* multiple GPUs on a single node
+* multiple nodes (each with potentially multiple GPUs)
+
+Both examples use **tf.distribute.Strategy** module, which is a part of **TensorFlow 2.0** and above.
+
```

Comment:
We should provide requirements. 
For example, running mnist.py gives me:
Traceback (most recent call last):
  File "mnist.py", line 2, in <module>
    import tensorflow_datasets as tfds
ModuleNotFoundError: No module named 'tensorflow_datasets'

Reply:
Requirements are now provided.

Reply:
OK